modules/cli/src/main/scala/scala/cli/commands/shared/SharedOptions.scala

```diff
@@ -180,6 +182,9 @@ final case class SharedOptions(
   @ValueDescription("/example/path")
   @Tag(tags.must)
     compilationOutput: Option[String] = None,
+  @HelpMessage("Add toolkit to classPath")
+  @ValueDescription("version|latest")
+    toolkit: Option[String] = None
```

Comment:
NIT to be consistent with `--with-compiler`
```suggestion
  @Name("toolkit")
    withToolkit: Option[String] = None
```